crates/bevy_render/src/camera/camera.rs

```diff
@@ -755,6 +755,17 @@ pub fn camera_system<T: CameraProjection + Component>(
                         }
                     }
                 }
+                if let Some(viewport) = &mut camera.viewport {
```

Comment:
Please add a comment to this block explaining why this check is needed. Linking this PR would be great as part of that :)